ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml

```diff
@@ -482,6 +487,10 @@ objects:
           exit 1
         fi
 
+        echo "JOB_NAME_HASH=${JOB_NAME_HASH}"
+        echo 'network:'
+        echo '  if: "br'$(printf ${LEASED_RESOURCE} | tail -c 3)'-'${JOB_NAME_HASH}'"'
+
```

Comment:
Shouldn't this info be printed out with the install-config.yaml?
If the install-config.yaml isn't already being printed out, can we do that instead?

Reply:
As discussed, I cated out the entire file...